vector/src/main/java/fr/gouv/tchap/activity/TchapLoginActivity.java

```diff
@@ -2136,20 +2255,20 @@ private void cancelEmailPolling() {
 
     /**
      * The user display name has been updated.
-     * @TODO remove this method as soon as the server is able to handle correctly the displayname
      *
      * @param logMessage the message to log
+     * @TODO remove this method as soon as the server is able to handle correctly the displayname
      */
     private void onDisplayNameUpdateDone(final String logMessage) {
```

Comment:
Note: This method can be removed
I forgot it

Reply:
done